scripts/cmd/_run_enceladus.cmd

```diff
@@ -665,29 +659,20 @@ EXIT /B 0
 EXIT /B 0
 
 :temp_log_file
-    IF %DATE_PARSING_TYPE%==parser1 (
-        ::25.06.2021
-        SET yyyy=%date:~-4%
-        SET MM=%date:~3,2%
-        SET DD=%date:~0,2%
-    )
-    IF %DATE_PARSING_TYPE%==parser2 (
-        ::Fri 06/25/2021
-        SET YYYY=%date:~10,4%
-        SET MM=%date:~4,2%
-        SET DD=%date:~7,2%
-    )
-    IF %DATE_PARSING_TYPE%==parser3 (
-        :: 2021-06-25
-        SET yyyy=%date:~0,4%
-        SET MM=%date:~5,2%
-        SET DD=%date:~8,2%
+    ::Date&Time parsing
+    FOR /F "skip=1 tokens=1-6" %%A IN ('WMIC Path Win32_LocalTime Get Day^,Hour^,Minute^,Month^,Second^,Year /Format:table') DO (
+        if "%%B" NEQ "" (
+            SET /A FDATE=%%F*10000+%%D*100+%%A
+            :: prefixing 1000000 to ensure the hours is two digit (starting with 0 before 10)
+            SET /A FTIME=1000000+%%B*10000+%%C*100+%%E
```

Comment:
This is a very interesting approach, but why not. I would have naturally attempted to created a _number-padding_ method, but this works well, too.

Reply:
I found it the easiest. Batch scripts are designed to manipulate strings, as I see it 😉